internal/k8s_watcher/watcher.go

```diff
@@ -1,4 +1,4 @@
-package k8s_watcher
+package k8swatcher
```

Comment:
No need to remove underscore because rule has been excluded, I think

Reply:
Came back to the option with underscore